.github/workflows/snyk.yml

```
@@ -0,0 +1,28 @@
+    - name: Setup pnpm
+      uses: pnpm/action-setup@v4
+      with:
+        version: 10.13.1
+    - name: Install dependencies
+      run: pnpm install --frozen-lockfile
+    - name: Run Snyk to check for vulnerabilities
+      uses: snyk/actions/node@master
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      with:
+        args: --severity-threshold=high
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)